config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "main": {
```

Comment:
Would be great to have a test for this but in the absence of this can you add a comment on what to supply here for this to work?

Reply:
@samvantran
Hmm, I don't know if comments are acceptable in JSON format. I'll just add a dummy one instead?